Source/View/Filter Collection View/FilterCollectionView.swift

```diff
@@ -30,7 +30,7 @@ public protocol FilterCollectionViewDelegate {
 public class FilterCollectionView: UIView, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
     
     public static let nibName: String = "FilterCollectionView"
-    
+    let AppDelegate = UIApplication.shared.delegate as! AppDelegate
```

Comment:
We shouldn't be using forced casting, it can easily break the application if someone tries to change something and aren't fully familiar with the work you did